docker-compose.yml

```
@@ -0,0 +1,64 @@
+services:
+  # PostgreSQL Database
+  postgres:
+    image: postgres:16-alpine
```

Using basic postgres here is simpler:
```suggestion
    image: postgres
```